### ISSUE TITLE
Servers can still get an A even if they can't handshake with most browsers

### ISSUE THREAD

**Issue by Pascal666** [ssllabs/ssllabs-scan#507]
Handshake Simulation results do not appear to affect a server's Overall Rating.

**Comment by ArchangeGabriel** [comment]
Yes, and I think this is partially right since getting maximum scores means you have to break support for most browsers. It’s even worse if you consider all NIST curves to be backdoored and choose to support only 25519 and/or 448.

**Comment by Pascal666** [comment]
Handshaking with older browsers probably shouldn't matter, but IMHO a server that cannot handshake with the current versions of Chrome, Firefox, and Android should not be given an A.  Perhaps the loss of a grade for each major browser whose current version the server cannot handshake with?

**Comment by ArchangeGabriel** [comment]
OK, then in this direction you’re likely already penalized for this: latest version of Chrome/Firefox/Android supports secure crypto, if you do not support them this very likely mean that you only support obsolete crytpo. Or do you have other ideas/example in mind? Especially a site that effectively scores an A but does not handshake with either Android, Chrome or Firefox in their latest version?

**Comment by Pascal666** [comment]
Example of a severely broken server receiving an A:
![screencapture-ssllabs-ssltest-analyze-html-1496223097318](https://cloud.githubusercontent.com/assets/1330708/26630052/c06f714c-45ca-11e7-8afb-8979b8ee3daf.png)

**Comment by ArchangeGabriel** [comment]
Hum. I see, Firefox/Chrome/Android do not support AES CBC with SHA-2, only SHA-1 (not sure why though). Well I would be in favour of moving SHA-1 suites support to a lower grade. But I’m not running SSLLabs.

Looking a bit around I found this: https://community.qualys.com/thread/12415. So I guess this is expected as a valid behaviour.

Lack of support in browsers is strange especially in the light of the `Modern` section in https://wiki.mozilla.org/Security/Server_Side_TLS.

**Comment by ivanr** [comment]
As you've correctly pointed out, interoperability is not taken into account at the moment. I will consider it for the next grading update. Thanks!
